spec/javascripts/Navigation/ActiveMenuTitle.spec.jsx

```diff
@@ -11,29 +11,28 @@ it('should return the proper title depending on the current menu', () => {
 
   expect(getWrapper('personal').text()).toEqual('Account Settings')
   expect(getWrapper('account').text()).toEqual('Account Settings')
+  expect(getWrapper('active_docs').text()).toEqual('Account Settings')
 
   expect(getWrapper('buyers').text()).toEqual('Audience')
   expect(getWrapper('finance').text()).toEqual('Audience')
   expect(getWrapper('cms').text()).toEqual('Audience')
   expect(getWrapper('site').text()).toEqual('Audience')
   expect(getWrapper('settings').text()).toEqual('Audience')
   expect(getWrapper('audience').text()).toEqual('Audience')
-
-  expect(getWrapper('applications').text()).toEqual('All APIs')
-  expect(getWrapper('active_docs').text()).toEqual('All APIs')
+  expect(getWrapper('applications').text()).toEqual('Audience')
 
   expect(getWrapper('serviceadmin', { name: 'Test' }).text())
-    .toEqual('Product: Test')
+    .toEqual('Products')
   expect(getWrapper('backend_api', { name: 'Test' }).text())
-    .toEqual('Backend: Test')
+    .toEqual('Backends')
 })
 
 it('should return the right title and icon when APIAP is disabled', () => {
   const wrapper = getWrapper('serviceadmin', { name: 'Test' }, false)
-  expect(wrapper.text()).toEqual('Api: Test')
-  expect(wrapper.find('i').first().prop('class')).toEqual('fa fa-puzzle-piece')
+  expect(wrapper.text()).toEqual('Products')
+  expect(wrapper.find('i').first().prop('class')).toEqual('fa fa-cubes')
 })
 
-it('should return a default title', () => {
-  expect(getWrapper().text()).toEqual('Choose an API')
+it('should not return a default title', () => {
```

Comment:
This means our default case will be returning an empty line?

Reply:
The default case is the ContextSelector "current section" would be empty, but the list would be still there and. However, this case should never happen.

Reply:
Safe to remove then, no?

Reply:
Yes indeed! Although it's there basically to make Flow (and I think Eslint) happy. Making the return type optional makes it unnecessarily verbose and what would be the result anyway? Not rendering the component? I think it's even more safe to at least have a failsafe and render it empty. WDYT?

Reply:
It's good enough I'd say. Maybe if needed, next iteration (if there's such a thing) we could refactor the entire switch case to a fn that takes a hash with the options.